pkg/console/operator/operator.go

```diff
@@ -54,7 +54,7 @@ const (
 )
 
 const (
-	minimumConsoleReplicas = 3
+	minimumConsoleReplicas = 2
```

Comment:
Should this just be called `consoleReplicas` since it will always be that value (at least for now).